lighthouse-cli/test/smokehouse/core-tests.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license Copyright 2019 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+import a11y from './test-definitions/a11y.js';
```

Comment:
we could drop this file and have it just run all the files in `test-definitions/` by default, similar to our `*-test.js` pattern for unit tests. It would need to be generated at build time for `yarn build-smokehouse-bundle` to replace it, though.